Traceback (most recent call last):
  File "<string>", line 40, in <module>
Exception: This is the error message
